docs/_includes/head.html

```diff
@@ -0,0 +1,170 @@
+<meta name="turbolinks-cache-control" content="no-cache" />
+<meta
+  name="viewport"
+  content="width=device-width, initial-scale=1, maximum-scale=1"
+/>
+<meta
+  property="og:image"
+  content="https://res.cloudinary.com/hy4kyit2a/image/upload/og-image.jpg"
+/>
+<link
+  rel="icon"
+  href="//developer.salesforce.com/resources2/favicon.ico"
```

Comment:
Shouldn't these image's href use `https` ?

Reply:
Doesn't really matter, right? That is without a protocol and the site always loads as HTTPS.

Reply:
I think that it only matters for the development scenario if the page is accessed directly without spinning up the server. For production, you are correct in that it will use the protocol being used by the site.

Reply:
Oh, i just realized. These are part of the generated pages from the docs team. They shouldn't be edited directly.

Reply:
Ok, that's good to know 👍